habitat-lab/habitat/core/env.py

```diff
@@ -128,6 +128,28 @@ def __init__(
             }
         )
         self.action_space = self._task.action_space
+
+        # Sharing controller between actions
+        # This should go somewhere else, but it is unclear where
+        # we want to share a controller across actions since the controller
+        # is stateful
+        from habitat_baselines.articulated_agent_controllers import (
+            HumanoidRearrangeController,
+        )
+
+        walk_pose_path = (
+            "data/humanoids/humanoid_data/walking_motion_processed.pkl"
```

Comment:
This looks strange to me. We rarely (never?) hard-code data paths in python code.

Reply:
Also, is this file needed for this code to run? How do users get it?

Reply:
Right! I will change it so that it is part of the humanoid config. The plan is to release this data with Habitat3.0 but there are some steps we need to do in order to release this first.

Reply:
Addressed in the latest commit